mediator/med_phases_prep_atm_mod.F90

```diff
@@ -295,4 +309,41 @@ subroutine med_phases_prep_atm_enthalpy_correction (gcomp, hcorr, rc)
 
   end subroutine med_phases_prep_atm_enthalpy_correction
 
+  !-----------------------------------------------------------------------------
+  subroutine med_phases_prep_atm_enthalpy_runoff(gcomp, hcorr, rc)
+
```

Comment:
Can here please be a comment on what this does and what it is for?
Is this called from nuopc_comp_ocn? I am not sure if I missed a call for this subroutine in the component PRs.